_rules/autocomplete-valid-73f2c2.md

```diff
@@ -19,7 +19,7 @@ authors:
 
 ## Applicability
 
-The rule applies to any HTML `input`, `select` and `textarea` element with an `autocomplete` attribute that is not empty (""), except if one of the following is true:
+The rule applies to any HTML `input`, `select` and `textarea` element with an `autocomplete` attribute that is neither empty ("") nor only [whitespace](#whitespace), except if one of the following is true:
```

Comment:
Editorial: I suggest we put this "is neither empty ("") nor only [whitespace](#whitespace)" bit in the list of exceptions, rather than have 1 inline exception, and 4 listed ones.